sympy/stats/frv.py

```diff
@@ -248,14 +242,17 @@ def __new__(cls, domain, density):
 
     def prob_of(self, elem):
         elem = sympify(elem)
-        return self._density.get(elem, 0)
+        density = self._density
+        if isinstance(list(density.keys())[0], FiniteSet):
```

Comment:
Horror. We should not make the code dependent on the order of elements of `dict`. The ordering of `dict` may vary on the Python version.

Reply:
Anyway, if it was already implemented like this, let's keep it this way.

Reply:
In this case all the keys are of the same type so order doesn't matter,  we can check any key from the list.